types/datadog-winston/datadog-winston-tests.ts

```diff
@@ -0,0 +1,10 @@
+import { DataDogTransport } from "datadog-winston";
```

Comment:
Fix this usage to resemble usage shown at https://github.com/itsfadnis/datadog-winston#usage